readthedocs/proxito/urls.py

```diff
@@ -151,7 +152,52 @@
     re_path(r"^(?P<path>.*)$", ServeDocs.as_view(), name="docs_detail"),
 ]
 
-urlpatterns = health_check_urls + proxied_urls + core_urls + docs_urls
+
+# Declare dummy "dashboard URLs" in El Proxito to be able to ``reverse()`` them
+# from API ``/_/addons/`` endpoint. Mainly for the the ``*.urls`` fields. We
+# cannot resolve ``*._links`` fields properly yet, but they are not required at
+# this point. We can come back later here if we need them.
+# See https://github.com/readthedocs/readthedocs-ops/issues/1323
+dummy_dashboard_urls = [
+    # /dashboard/<project_slug>/
+    re_path(
+        r"^(?P<project_slug>{project_slug})/$".format(**pattern_opts),
+        TemplateView.as_view(template_name="proxied_view.html"),
```

Comment:
Does this exist?

Reply:
No. This view should never hit by the user since we don't expose it to them.

However, thinking a bit more about this, I think we should probably return "I'm a teapot" here just in case. What do you think?

Reply:
I implemented this in [`c7238db` (#11227)](https://github.com/readthedocs/readthedocs.org/pull/11227/commits/c7238dbf9e101a1b5315af98136628e09839be05)

It looks like this.

![Screenshot_2024-03-19_12-13-52](https://github.com/readthedocs/readthedocs.org/assets/244656/0fe83ae9-6d2a-4c3b-84ec-a55a677f1aed)

I think this approach is better than 500 in case somebody somehow hits those URLs 😄